### PR TITLE
Aswathy/eslint prettier rules

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Remove Prettier config and cover the rules in Eslint package

### CODE REVIEW

**Review by markw-deriv — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -98,106 +103,114 @@ module.exports = {
...
+          ["^.+\\.?(css)$"],
         ],
       },
     ],
-  
   },
-  
+  "linebreak-style": ["error", "unix"],
+  "jsx-quotes": ["error", "prefer-single"],
+  "no-confusing-arrow": ["error", { allowParens: false }],
```
Comment (shayan-deriv) [inline index.js:208]: do we need this? 🤔 it will throws lots of error with the current code we have in both and deriv-app I guess.

**Review by shayan-deriv — COMMENTED** [review]
The rest LGTM.

File: index.js (hunk 1 above)
Comment (shayan-deriv) [reply index.js:208]: just ignore this comment

**Review by shayan-deriv — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (aswathy-deriv) [reply index.js:208]: Thanks @shayan-deriv

**Review by ako-deriv — APPROVED** [review]

**Pull request merged**